addons/send/send.lua

```diff
@@ -4,10 +4,24 @@ _addon.command = 'send'
 _addon.author = 'Byrth'
 
 windower.register_event('addon command',function (...)
-    local term = table.concat({...}, ' ')
+	local term = table.concat({...}, ' ')
+	
+	local target = windower.ffxi.get_mob_by_target('t')
```

Comment:
This function (as well as the one below with `lastst`) are expensive. You're calling them on every send command. Instead I'd use `gsub` with a replacement function:
```lua
term = term:gsub('<(%a+)id>', function(target_string)
    local entity = windower.ffxi.get_mob_by_target(target_string)
    return entity and entity.id or '<' .. target_string .. 'id>'
end)
```

This way it'll automatically work with everything that `get_mob_by_target` works with, like `<meid>`, `<htid>`, `<stid>`, and so on. It would, however, mean that  `<lstid>` would not work, it would be `<laststid>` instead, which is something I'd suggest to do anyway to keep consistency.

Then you could also remove the block below.